src/System/Router/ErrorBoundary.tsx

```diff
@@ -41,6 +41,11 @@ export class ErrorBoundary extends React.Component<Props, State> {
   }
 
   static getDerivedStateFromError(error: Error | HttpError) {
+    // This adds a useful stacktrace that can make debugging easier when writing tests
+    if (window.__TEST__) {
```

Comment:
I (think) you can use jest's provided env variable! so something like `process.env.JEST`. Its there, but not entirely sure how to get at it. Maybe even `getENV('JEST')` could work?

Reply:
Also we might need to check for SSR here, if needing to use window -- something like `typeof window !== undefined && window.TEST`, if the JEST approach doesn't work.